### ISSUE TITLE
Code Review

### ISSUE THREAD

**Issue by obenland** [coreymckrill/nearby-wordpress-events#53]
- Is there any way we can avoid [using ids in CSS](https://github.com/coreymckrill/nearby-wordpress-events/blob/master/css/dashboard.css#L11)?
- Function/class/method documentation need `@since` and `@access` docs
- There shouldn't be blank line between `@param` and `@return`
- Missing information about default values when params are optional
- Missing `@return` descriptions.
- Maybe `get_request_url` instead of `build_api_request_url`? ([#](https://github.com/coreymckrill/nearby-wordpress-events/blob/master/includes/class-wp-nearby-events.php#L63))
- [Inline comments](https://github.com/coreymckrill/nearby-wordpress-events/blob/master/includes/class-wp-nearby-events.php#L96) should end with a period.
- Can we use `get_option( 'date|time_format' )` [here](https://github.com/coreymckrill/nearby-wordpress-events/blob/master/includes/class-wp-nearby-events.php#L254)?
- It seems easier to just use `wp_localize_script()` [here](https://github.com/coreymckrill/nearby-wordpress-events/blob/master/includes/script-loader.php#L26).
- Feels weird to use camelCase notation in PHP [here](https://github.com/coreymckrill/nearby-wordpress-events/blob/master/includes/dashboard.php#L33). Could we just use the translation strings directly here, without getting them from the JS output?
- String localization are usually in a variable called `l10n` rather than `i18n` it seems. ([#](https://github.com/coreymckrill/nearby-wordpress-events/blob/master/includes/script-loader.php#L46))

**Comment by iandunn** [comment]
> Is there any way we can avoid using ids in CSS?

ae4eb79 removes a few. The remaining ones are places where the IDs are generated by Core, and corresponding classes aren't available. There are a few potential workarounds, but they seem pretty awkward, so I'm not sure they're worth it. 

I'm not clear on the exact reason for avoiding IDs, though. There's no mention of it in the CSS style guide.

**Comment by obenland** [comment]
> I'm not clear on the exact reason for avoiding IDs, though. There's no mention of it in the CSS style guide.

Yeah that's not necessarily a WP coding guideline as much as a general CSS best practice to avoid selector specificity wars down the line

**Comment by iandunn** [comment]
> Function/class/method documentation need since and access docs
> There shouldn't be blank line between param and return
> Missing information about default values when params are optional
> Missing return descriptions.
> Inline comments should end with a period.

Fixed in e85c213.

**Comment by iandunn** [comment]
> Maybe get_request_url instead of build_api_request_url? (#)

Fixed in 	73b3b65

> Can we use get_option( 'date|time_format' ) here?

it's important to have the day of the week so that people know if they can attend or not, without having to open the link.

We can use the configured time format, though. Fixed that in 	9f744bc

**Comment by iandunn** [comment]
> It seems easier to just use wp_localize_script() here.

Ah, ok. I had originally assumed that `add_inline_script_data()` would be more appropriate, since the data includes more than just internationalized strings.

Fixed in f43ef51

> Feels weird to use camelCase notation in PHP here. 

I did that because they're also used in JavaScript, so the conventions conflict. I'm happy to use the PHP one, though.

Fixed in 32afd9e

> Could we just use the translation strings directly here, without getting them from the JS output?

It wouldn't be DRY, so any changes to them would have to be manually synced, but I'm happy to make that change if you think it's a good one.

> String localization are usually in a variable called l10n rather than i18n it seems. (#)

Fixed in 32afd9e
